stable/awsebscsiprovisioner/templates/roles.yaml

```diff
@@ -187,6 +187,9 @@ rules:
   - apiGroups: ["snapshot.storage.k8s.io"]
     resources: ["volumesnapshots/status"]
     verbs: ["update"]
+  - apiGroups: ["apiextensions.k8s.io"]
+    resources: ["customresourcedefinitions"]
+    verbs: ["create"]
```

Comment:
Do we not need the other verbs like in https://github.com/mesosphere/charts/blob/2336e7191f0a62d189fd88450b3ea3d9ef465101/stable/awsebscsiprovisioner/templates/roles.yaml#L141?

Reply:
+1. yes it is weird we need create but not other verbs

Reply:
I only saw an error with creating a CRD, not listing etc. And this change was sufficient to fix it. I'll add the rest though, shouldn't hurt.

Reply:
Maybe delete on cleanup?